### PR TITLE
feat(internal): fix changelogs

### PR DESCRIPTION
**Pull request by viceice**
* fix missing paging
* fix github fetch all file contents (first 1000 files from repo root)
* use optimized api's

### CODE REVIEW

**Comment by viceice** [comment]
real tests succeeded

File: lib/datasource/metadata.ts [hunk 1]
```diff
@@ -34,6 +34,12 @@ const manualChangelogUrls = {
     requests: 'https://github.com/psf/requests/blob/master/HISTORY.md',
     wagtail: 'https://github.com/wagtail/wagtail/tree/master/docs/releases',
   },
+  docker: {
+    'gitlab/gitlab-ce':
+      'https://gitlab.com/gitlab-org/omnibus-gitlab/-/blob/master/CHANGELOG.md',
+    'gitlab/gitlab-runner':
+      'https://gitlab.com/gitlab-org/gitlab-runner/-/blob/master/CHANGELOG.md',
+  },
```
Comment (viceice) [inline lib/datasource/metadata.ts:42]: could be move to separate pr

File: lib/workers/pr/changelog/github.spec.ts [hunk 2]
```diff
@@ -28,7 +29,7 @@ const upgrade: BranchUpgradeConfig = {
   ],
 };
 
-describe('workers/pr/changelog', () => {
+describe(getName(__filename), () => {
```
Comment (viceice) [inline lib/workers/pr/changelog/github.spec.ts:32]: can do this in seperate pr

**Comment by viceice** [comment]
ready for review

**Review by rarkins — APPROVED** [review]

**Review by rarkins — APPROVED** [review]
